src/config.js

```diff
@@ -1,6 +1,7 @@
 export const DEFAULT_STARTER = 'graasp/graasp-app-starter-react';
 export const DEFAULT_FRAMEWORK = 'react';
 export const DEFAULT_PATH = './';
+export const BUILD_DIRECTORY = '/build';
```

Comment:
Not sure if this is needed here and why it's absolute?

Reply:
It was part of the other PR, I'll move it.